src/controllers/update/update.review.beneficial.owner.individual.ts

```diff
@@ -0,0 +1,34 @@
+import { UPDATE_BENEFICIAL_OWNER_BO_MO_REVIEW_URL, UPDATE_BENEFICIAL_OWNER_TYPE_URL, UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE } from "../../config";
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData } from "../../utils/application.data";
+import { logger } from "../../utils/logger";
+
+export const get = (req: Request, res: Response) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
+  const appData = getApplicationData(req.session);
+  const index = req.query.index;
+
+  let dataToReview = {};
+  if (appData?.beneficial_owners_individual){
+    dataToReview = appData?.beneficial_owners_individual[Number(index)];
+  }
+
+  return res.render(UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE, {
+    backLinkUrl: UPDATE_BENEFICIAL_OWNER_BO_MO_REVIEW_URL,
+    templateName: UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_PAGE,
+    ...dataToReview,
+    isOwnersReview: true
```

Comment:
what is this boolean for?

Reply:
Is used to for contents display specific for review. This can easily be used also for the other bo type pages